src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -108,6 +108,14 @@ CREATE TABLE "trial_data" (
   created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
 );
 
+CREATE TABLE "trial_data_file" (
+  trial_file_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  trial_data_id UUID REFERENCES "trial_data" (trial_data_id),
```

Comment:
Couldn't the `file` table directly reference `trial_data`? Not sure if there's other places we would upload files.

Reply:
we could, but imo this adds very complexity 🤷 

some places where we might also use `file` in the future:

- custom user avatars
- uploaded audio notes